Userland/Games/Chess/ChessWidget.h

```diff
@@ -140,7 +139,7 @@ class ChessWidget final
     Color m_marking_secondary_color { Color::from_argb(0x6655dd55) };
     Chess::Color m_side { Chess::Color::White };
     HashMap<Chess::Piece, RefPtr<Gfx::Bitmap const>> m_pieces;
-    DeprecatedString m_piece_set;
+    StringView m_piece_set;
```

Comment:
I wasn't very clear, but my comment is also valid for this `StringView`. Keeping a `StringView` to some data that you don't own can be quite silly.
This field isn't used anywhere, so you can delete it. Btw that can be in its own commit.